crow/setup.py

```diff
@@ -12,8 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from distutils.core import setup, Extension
```

Comment:
Do we need to keep this import given your changes?

Reply:
Yes, setup is still being called and still uses Extension.  What is broken is that setup.py no longer supports --install-platlib

Reply:
(At some point we may need to switch entirely away from setup.py  https://pip.pypa.io/en/stable/reference/build-system/setup-py/ )

Reply:
Sounds good to me. Let's keep this modification in our radar for the near future.